packages/astro/src/types/public/config.ts

```diff
@@ -2083,6 +2083,18 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		 */
 		clientPrerender?: boolean;
 
+		/**
+		 *
+		 * @name experimental.failOnPrerenderConflict
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 5.x
+		 * @description
+		 * When two routes generate the same prerendered URL, fail the build instead of silently skipping one.
```

Comment:
```suggestion
		 * When two routes generate the same prerendered URL, fail the build instead of skipping one.
```
It's no longer silent!